extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -183,6 +161,10 @@ export class DeployConfigPage extends DacFxConfigPage {
 			return false;
 		}
 		let values = await this.getDatabaseValues();
+		// If there are no databases, call the disableUpgradeRadioButton function.
+		if (values.length === 0) {
+			this.disableUpgradeRadioButton();
+		}
```

Comment:
can an else be added to set this.upgradeRadioButton.enabled = true here? Just to make sure it gets enabled again if the selected server gets changed to one with databases

Reply:
Acknowledged, I will add another helper function, enableUpgradeRadioButton which will enable the upgrade radio button in the else condition.